From 724c9436a12539ab9afe164f538c24792e48a133 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Mon, 10 Sep 2012 22:49:52 +0000
Subject: [PATCH 1192/1357] MR1: HADOOP-8781. hadoop-config.sh should add JAVA_LIBRARY_PATH to LD_LIBRARY_PATH. (tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1@1383145 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 60c49a8e938219585c42ad167d040a0431fb9fa5)

Conflicts:

	CHANGES.txt
---
 bin/hadoop |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/bin/hadoop b/bin/hadoop
index d120d4c..9bed6b0 100755
--- a/bin/hadoop
+++ b/bin/hadoop
@@ -336,6 +336,7 @@ HADOOP_OPTS="$HADOOP_OPTS -Dhadoop.id.str=$HADOOP_IDENT_STRING"
 HADOOP_OPTS="$HADOOP_OPTS -Dhadoop.root.logger=${HADOOP_ROOT_LOGGER:-INFO,console}"
 if [ "x$JAVA_LIBRARY_PATH" != "x" ]; then
   HADOOP_OPTS="$HADOOP_OPTS -Djava.library.path=$JAVA_LIBRARY_PATH"
+  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$JAVA_LIBRARY_PATH
 fi  
 HADOOP_OPTS="$HADOOP_OPTS -Dhadoop.policy.file=$HADOOP_POLICYFILE"
 
-- 
1.7.0.4

